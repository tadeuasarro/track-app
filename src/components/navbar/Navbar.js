import './navbar.css';

const Navbar = () => (
  <div className="navbar">
    <h1>Personal Expenses</h1>
  </div>
);

export default Navbar;
