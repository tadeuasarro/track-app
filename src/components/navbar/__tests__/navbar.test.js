import { render, screen, act } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Navbar from '../Navbar';

describe('Rendering component', () => {
  it('creates an App component', () => {
    act(() => {
      render(
        <Navbar />,
      );
    });
    // eslint-disable-next-line
    const element = screen.getByText('Personal Expenses');
  });
});

describe('Display', () => {
  it('renders correctly', () => {
    const comp = renderer.create(
      <Navbar />,
    ).toJSON();
    expect(comp).toMatchSnapshot();
  });
});
