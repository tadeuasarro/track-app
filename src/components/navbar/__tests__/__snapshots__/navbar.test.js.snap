// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display renders correctly 1`] = `
<div
  className="navbar"
>
  <h1>
    Personal Expenses
  </h1>
</div>
`;
